packages/services/f-analytics/src/mixins/_tests/analytics.methods.test.js

```diff
@@ -13,50 +12,6 @@ import analyticsMixin from '../analytics.mixin.vue';
 const localVue = createLocalVue();
 localVue.use(Vuex);
 
-const mockAuthTokenRegistered = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.'
-+ 'eyJlbWFpbCI6ImpvZS5ibG9nZ3NAanVzdGVhdHRha2Vhd2F5LmNvbS'
-+ 'IsImNyZWF0ZWRfZGF0ZSI6IjIwMjEtMDItMDhUMTA6Mjc6NDkuMTkz'
-+ 'MDAwMFoiLCJuYW1lIjoiSm9lIEJsb2dncyIsImdsb2JhbF91c2VyX2lkI'
-+ 'joiVTdOUkFsV0FnNXpPZHNkUmdmN25rVHlvaTkwWEVvPSIsImdpdmVuX25h'
-+ 'bWUiOiJKb2UiLCJmYW1pbHlfbmFtZSI6IkJsb2dncyIsImlhdCI6MTYxNTQ2OTUxNn0.VapH6uHnn4lHIkvN_mS9A9IVVWL0YPNE39gDDD-l7SU';
-
-const mockAuthTokenGuest = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.'
-+ 'eyJlbWFpbCI6ImpvZS5ibG9nZ3NAanVzdGVhdHRha2Vhd2F5LmNvbSIsImNyZ'
-+ 'WF0ZWRfZGF0ZSI6IjIwMjEtMDItMDhUMTA6Mjc6NDkuMTkzMDAwMFoiLCJuYW'
-+ '1lIjoiSm9lIEJsb2dncyIsImdsb2JhbF91c2VyX2lkIjoiVTdOUkFsV0FnNXp'
-+ 'PZHNkUmdmN25rVHlvaTkwWEVvPSIsImdpdmVuX25hbWUiOiJKb2UiLCJmYW1p'
-+ 'bHlfbmFtZSI6IkJsb2dncyIsInN1YiI6IjEyMzQ1Iiwicm9sZSI6Ikd1ZXN0Ii'
-+ 'wiaWF0IjoxNjE1NDY5NTE2fQ.ngfAKpiMH4Gk0Y4gAVC4KeLadWFtVXx4hD1_BSW9SN0';
-
-const userIdFromCookie = 'fjdhskgshjgk';
-
-const userDataWithMockAuthTokenRegistered = {
-    'a-UserId': userIdFromCookie,
-    authType: 'Login',
-    email: '1a9a31f72fbb57efd148bbfe06c169b97f6868200b422a5ae7fed7e3f853002a',
-    globalUserId: 'U7NRAlWAg5zOdsdRgf7nkTyoi90XEo=',
-    signinType: 'Email',
-    signupDate: '2021-02-08T10:27:49.1930000Z'
-};
-
-const userDataWithMockAuthTokenGuest = {
-    'a-UserId': userIdFromCookie,
-    authType: 'Login',
-    email: '1a9a31f72fbb57efd148bbfe06c169b97f6868200b422a5ae7fed7e3f853002a',
-    globalUserId: 'U7NRAlWAg5zOdsdRgf7nkTyoi90XEo=',
-    signinType: 'Guest',
-    signupDate: '2021-02-08T10:27:49.1930000Z'
-};
-
-const userDataWithoutMockAuthToken = {
-    'a-UserId': userIdFromCookie,
-    authType: undefined,
-    email: undefined,
-    globalUserId: undefined,
-    signinType: undefined,
-    signupDate: undefined
-};
-
 const mockAuthToken = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.'
```

Comment:
This `mockAuthToken` is the same as `mockAuthTokenRegistered` as I believe Andy copied it from my userData PR for the pageData test.